side-products/merkl/README.md

```diff
@@ -37,10 +37,10 @@ The script is ran regularly and for specific periods of time each time. This mea
 Precisely speaking, for a given pool with two tokens (A and B), the script looks into the swaps that took place on the pool during the period for which it is ran and computes a reward score for each position according to:
 
 - the fees earned by the position during the period, which represent the liquidity of the position used by the pool
-- the share of token A held by the position during swaps on the pool
-- the share of token B held by the position during swaps on the pool
+- the share of token A held by the position during swaps on the pool compared to the total amount of token A in the pool
```

Comment:
in this case: it should be: the **amount** of token A held by the position (if you compare it to the total amount in the pool)